autoextract/aio/client.py

```diff
@@ -114,19 +233,33 @@ async def request():
                 # good response
```

Comment:
it is not necessarily good, as `request_processor.process_results` may also raise an exception

Reply:
That's true. I'm just removing this comment for now.